codegen_glibmm/templates/stub.h.templ

```diff
@@ -25,6 +25,11 @@ public:
                           const Glib::ustring &object_path);
     void unregister_object();
 
+    int usage_count() const
```

Comment:
WIth inline methods, we put the `{` in the same line.

Reply:
Argh! Sorry, fixed.